custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -57,8 +58,13 @@ def configure(self, params):
             )
 
         model_metadata = read_model_metadata_yaml(self._code_dir)
-        self._schema_validator = (
-            SchemaValidator(model_metadata.get("typeSchema", {})) if model_metadata else None
+
+        # Set the environment variable STRICT_VALIDATION to use validation
+        print(os.getenv("STRICT_VALIDATION", False) in [1, "True", "true", "TRUE"])
```

Comment:
I don't think this is good, but I don't know what it is.

Reply:
debug, removed